src/createRoom.ts

```diff
@@ -268,6 +268,9 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                 metricsTrigger: "Created",
             });
         }
+        if (opts.roomType === RoomType.Space) {
+            SpaceStore.instance.setActiveSpace(roomId, false);
+        }
```

Comment:
hmm. I'm still a bit concerned that the `Action.ViewRoom` dispatch above this isn't working properly. When the user clicks on spaces we send out the same dispatch, so we know that it's meant to work. We also set `andView` on the options when creating the space, signifying that the dispatch should be sent.

This sort of indicates that the fix will be somewhere else in the dispatch handling pipeline rather than needing to call the SpaceStore specifically.